JitsiConference.js

```diff
@@ -156,7 +157,12 @@ export default function JitsiConference(options) {
     this.eventEmitter = new EventEmitter();
     this.options = options;
     this.eventManager = new JitsiConferenceEventManager(this);
-    this.participants = {};
+
+    /**
+     * List of all the participants in the conference.
+     * @type {Map<string, JitsiParticipant>};
+     */
+    this.participants = new Map();
```

Comment:
Out of curiosity why did you change it to `Map`?

Reply:
A Map made more sense for the participant list, also it is going to make the transition to TS easier. I am trying to make small incremental changes whenever I am touching a class so that its easy to make the switch later.

Reply:
I don't see how it could affect the transition to TS but I see how a map can be more clear as a type.